Formula/jsign.rb

```diff
@@ -0,0 +1,51 @@
+class Jsign < Formula
+  desc "Tool for signing Windows executable files, installers and scripts"
+  homepage "https://ebourg.github.io/jsign/"
+  url "https://github.com/ebourg/jsign/archive/refs/tags/4.2.tar.gz"
+  sha256 "653ce9626969ddcbf3365dee54c25c53586bdf3f5d9a067991e07121dd127bee"
+  license "Apache-2.0"
+  head "https://github.com/ebourg/jsign.git", branch: "master"
+
+  depends_on "maven" => :build
+  depends_on "openjdk@17" # The build fails with more recent JDKs
+
+  resource "testdata" do
+    url "https://github.com/ebourg/jsign/archive/refs/tags/4.2.tar.gz"
+    sha256 "653ce9626969ddcbf3365dee54c25c53586bdf3f5d9a067991e07121dd127bee"
+  end
```

Comment:
can you move this into L34 (inside `test` block)?

Reply:
actually isn't this the same thing of the artifact, you can use `pkgshare.install` to install the test files.

Reply:
I've moved the resource into the test block. I don't understand the benefit of using `pkgshare.install` though. It seems to copy the test files on the user's system when installing the package, but these files are neither examples nor templates, they have no value to an end user and aren't meant to be installed.